cvat/apps/engine/permissions.py

```diff
@@ -74,6 +88,26 @@ def extend_resource_with_rq_job_details(self, data: dict[str, Any]) -> None:
             }
         }
 
+class ExportableResourceExtension:
+    location: Location
+
+    @classmethod
+    def update_scope_params(
+        cls: Type[OpenPolicyAgentPermission],
```

Comment:
Why override the type here?

Reply:
As this class is a mixin, the type of cls cannot be determined automatically. This declaration enables code editor support for ones who need to use some cls methods.